backend/canvas_app_explorer/serializers.py

```diff
@@ -10,7 +14,16 @@ class Meta:
 class LtiToolSerializer(serializers.ModelSerializer):
     # Addiitonal expanded ead only field to make it easier on the front-end
     canvas_placement_expanded = CanvasPlacementSerializer(read_only=True, many=True, source="canvas_placement")
+    enabled = serializers.SerializerMethodField()
 
     class Meta:
         model = models.LtiTool
         fields = '__all__'
+
+    def get_enabled(self, obj: models.LtiTool) -> Union[bool, None]:
```

Comment:
Something I'd like to get better about (again) is using docstrings. I used these consistently in Java and more consistently in MyLA but I think not as much here. There's an [extension for this ](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) but I'm not sure the format. to use. Sphinx and "one-line Sphnix" seem the most popular but Google looks easiest to read, though taking up more space. 

We can go back and do this on another issue too, or start with this one!